src/core_plugins/kibana/public/management/sections/objects/_objects.js

```diff
@@ -206,11 +206,28 @@ uiModules.get('apps/management')
                       err.message = `Importing ${obj.title} (${obj.id}) failed: ${err.message}`;
                       notify.error(err);
                     });
-                })
-                .then(refreshIndex)
-                .then(refreshData)
-                .catch(notify.error);
+                });
+            }
+
+            const docTypes = docs.reduce((types, doc) => {
```

Comment:
Slick, but that only allows me to create 2 collections. That works for now, but I'm planning to add more exports, and order will become important in those cases too. This structure makes it easy to add more ordered collections.